ExamPortal/Models/question.js

```diff
@@ -0,0 +1,28 @@
+const mongoose  = require('mongoose')
+
+const Schema = mongoose.Schema
+
+const questionSchema = new Schema({
+	qHeading: String,
+	qText: String,
+	qNo: Number,
+	ans: String,
+	options:{
+		op1: String,
+		op2: String,
+		op3: String,
+		op4: String
+	},
+	weightage: Number,
+	time: {
+		type: Date,
+		default: Date.now
+	},
+	examCode: String
```

Comment:
It should be `_id` of examSchema, which indeed would be an ObjectID

Reply:
We have created a unique exam code which will be test key for student. we discussed it with Vishal too.
Birendra here